Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -7072,4 +5788,194 @@ public void Actions_CanIterateOverActionsInMap()
 
         Assert.That(map.ToList(), Is.EquivalentTo(new[] { action1, action2, action3 }));
     }
+
```

Comment:
The refactor of this file above makes it incredibly hard to review (and to find the interesting bits below). Can you try to do refactors like that in separate PRs in the future for the sake of reviewability?

Reply:
I'll try.

This refactor I made because CoreTests_Actions.cs reached the size limit on code analysis in Rider and I couldn't figure out how to increase the limit. So I split the file into three. Didn't anticipate that it would result in such a clusterfuck of a diff.

Reply:
I guess the way to do it then is to PR a separate branch where you move the tests, and merge that back into your branch. (But, yes, it is a hassle)

Reply:
And when you are two weeks late with your promised changes, it's the kind of stuff that tends to be the first to land on the "ooops, sorry" list :D

Reply:
Hmm, looking at the fallout, only now realize how intrusive that splitting up of files actually is. Didn't think much about it when I did it.